src/main/webapp/js/instructorFeedbackEdit/feedbackPath.js

```diff
@@ -0,0 +1,70 @@
+$(document).ready(function() {
+    FeedbackPath.attachEvents();
+});
```

Comment:
Can this be moved to the js file specifically for the instructor feedback edit page?
That way, we can unit test the `FeedbackPath` variable on its own without having to worry about the document.ready even being called. I actually want to implement this kind of js paradigm across the code base in the near future.